ui/src/utils/actions.js

```diff
@@ -36,24 +36,51 @@ const moveIdentity = (fromUuid, toUuid, action, dialog) => {
 };
 
 const groupIdentities = identities => {
-  const icons = ["git", "github", "gitlab"];
+  const icons = [
+    { source: "git", icon: "mdi-git" },
+    { source: "github", icon: "mdi-github" },
+    { source: "gitlab", icon: "mdi-gitlab" },
+    { source: "dockerhub", icon: "mdi-docker" },
+    { source: "jira", icon: "mdi-jira" },
+    { source: "rss", icon: "mdi-rss" },
+    { source: "slack", icon: "mdi-slack" },
+    { source: "stackexchange", icon: "mdi-stack-exchange" },
+    { source: "telegram", icon: "mdi-telegram" },
+    { source: "twitter", icon: "mdi-twitter" }
+  ];
+  // Group identities by data source
   const groupedIdentities = identities.reduce((result, val) => {
-    let source = val.source.toLowerCase();
-    if (!icons.find(icon => icon === source)) {
-      source = "others";
+    let source = val.source.toLowerCase().replace(/\s+/g, "");
+    const sourceIcon = icons.find(icon => icon.source === source);
+    if (!sourceIcon) {
+      source = "Other sources";
     }
     if (result[source]) {
       result[source].identities.push(val);
     } else {
       result[source] = {
         name: source,
-        identities: [val]
+        identities: [val],
+        icon: sourceIcon ? sourceIcon.icon : "mdi-account-multiple"
       };
     }
     return result;
   }, {});
 
-  return Object.values(groupedIdentities);
+  // Sort identities by alphabetical order
+  let sortedIdentities = Object.values(groupedIdentities).sort((a, b) =>
+    a.name.localeCompare(b.name)
+  );
+
+  // Move "other" identities to end of list
+  sortedIdentities.push(
+    ...sortedIdentities.splice(
+      sortedIdentities.findIndex(identity => identity.name == "Other sources"),
```

Comment:
"Other sources" is used at least twice in the code. I think it would be better to assign it to a constant and reference the constant instead of the string.